src/test/java/seedu/address/model/person/TutorialGroupTest.java

```diff
@@ -25,11 +25,13 @@ public void isValidTutorialGroup() {
         assertThrows(NullPointerException.class, () -> TutorialGroup.isValidTutorialGroup(null));
 
         // invalid tutorial group
-        assertFalse(TutorialGroup.isValidTutorialGroup("B")); // contains character
+        assertFalse(TutorialGroup.isValidTutorialGroup("B")); // one character only
+        assertFalse(TutorialGroup.isValidTutorialGroup("09")); // two digits only
+        assertFalse(TutorialGroup.isValidTutorialGroup("T124")); // length exceeds maximum
+        assertFalse(TutorialGroup.isValidTutorialGroup("t09")); // lowercase character
```

Comment:
was thinking maybe we could make it case insensitive hahah, more convenient right?